lib/hooks.ts

```diff
@@ -75,3 +75,24 @@ export const useOrder = (orderId: number) => {
         error,
     };
 }
+
+export const useShippingAndProductsOrder = (orderId: number) => {
+    const { context } = useSession();
+    const params = new URLSearchParams({ context }).toString();
+    const shouldFetch = context && orderId !== undefined;
+
+    const { order, error: orderError } = useOrder(orderId);
```

Comment:
since `useOrder` is already a hook, why not just call directly where we need it in `pages`, rather than adding additional complexity here?